src/format.rs

```diff
@@ -1023,6 +1023,8 @@ impl Documentable for &UntypedClauseGuard {
 
             ClauseGuard::Float { value, .. } => value.to_string().to_doc(),
 
+            ClauseGuard::Tuple { elems, ..} => elems.join(", ").to_doc(),
```

Comment:
This renders invalid Gleam syntax and doesn't indent. We'll want to use use the pretty printing algebra. A good reference would be to see how we format UntypedExpr::Tuple in this module.